tests/FSharp.Compiler.ComponentTests/Language/InterfaceTests.fs

```diff
@@ -4,8 +4,8 @@ open Xunit
 open FSharp.Test.Compiler
 
 [<Fact>]
-let ``Concrete instance method is not allowed in interfaces in lang preview``() =
-    FSharp $"""
+let ``Concrete instance method is not allowed in interfaces in lang version80``() =
```

Comment:
Thank you for taking care of this, we do forget to do that from time to time.

Reply:
@vzarytovskii, the current behavior would be unexpected, but yes, it would change the behavior of existing applications if they relied on the bug.  I think we would take this as a breaking change.  It's not actually clear to me that the INumerics would work as expected without this fix.

For example:
![image](https://github.com/dotnet/fsharp/assets/5175830/cdc6bf59-5893-4ebc-ad71-dc61e5547883)

Someone should probably take a look, perhaps I should.

Reply:
I was trying to implement `INumber.Max`, and came across this bug.  I was not able to get F# to call my defined `Max`, instead the base method keeps getting called.  The unit tests from [here](https://gitlab.com/dbnZA/Jita/-/merge_requests/67) pass with this change, but not without.